projects/1.18/assets/tiny-redstone/tinyredstone/lang/zh_cn.json

```
@@ -1 +1,96 @@
-{}
\ No newline at end of file
+{
+  "key.tinyredstone.rotation_lock": "锁定旋转（键位+滚轮）",
+  "block.tinyredstone.redstone_panel": "红石板",
+  "block.tinyredstone.tiny_container": "微型元件容器",
+  "message.item.redstone_panel": "§7用于放置微型红石元件的面板。\n使用染料右击来改变颜色。§r",
+  "message.item.redstone_wrench": "§7右击红石板使其旋转。\n左击移除微型红石元件。\nShift+左击移除所有微型红石元件。§r",
+  "message.item.redstone_panel_cell": "可以被放置在红石板或者世界中。",
+  "message.item.tiny_color_selector": "§7右击微型方块来选择其颜色。\n兼容绝大多数方块型元件。§r",
+  "item.tinyredstone.tiny_redstone": "微型红石粉",
+  "message.item.tinyredstone.tiny_redstone": "传输红石信号，与原版红石粉类似。\n右击选择要连接的面。",
+  "item.tinyredstone.tiny_repeater": "微型中继器",
+  "message.item.tinyredstone.tiny_repeater": "后方接收到红石信号时，向前输出强度为15的强充能红石信号，可用右键调节延迟（1~4红石刻）。",
+  "item.tinyredstone.tiny_redstone_torch": "微型红石火把",
+  "message.item.tinyredstone.tiny_redstone_torch": "可以作为红石信号源和非门。\n像原版火把一样附着在微型方块上，但当位于面板或方块顶部时，可以向远离你的方向倾斜。\n因此无需将其放在方块上便可制作非门。",
+  "item.tinyredstone.redstone_wrench": "微型元件板手",
+  "item.tinyredstone.tiny_comparator": "微型比较器",
+  "message.item.tinyredstone.tiny_comparator": "当放置在面板边缘时，将从其毗邻的方块和容器比较器输出中接收弱信号。\r右击切换为作差模式。",
```

“弱信号”是什么？

---

wiki里没有weak signal相关解释，我对着弱充能翻的

---

```suggestion
  "message.item.tinyredstone.tiny_comparator": "从相邻的方块获取弱充能信号。如放置在面板边缘，还可读取相应容器的比较器输出。\r右击切换为作差模式。",
```
我知道了，应该是断句不对
原文：Will pick up *weak signals from adjacent blocks* and *inventory comparator outputs __when placed on the edge of a panel__*.\rRight click to toggle subtract mode.

---

什么魔鬼断句我超